wiki/en/Software-Manual.md

```diff
@@ -268,30 +322,20 @@ Increases the gain from your device. Use this if your device delivers a gain tha
 
 ### Feedback Protection
 
-Attempts to detect audio feedback loops or loud noise in the first three seconds after you connected to a server. Once detected, this feature will show a message and activate the "Mute Myself" button to mute you in your own mix. 
+Attempts to detect audio feedback loops or loud noise in the first three seconds after you connected to a server. Once detected, this feature will show a message and activate the "Mute Myself" button to mute you in your own mix.
 
 ### Input Balance
 
 Controls the relative levels of the left and right local audio channels. For a mono signal
 it acts as a pan between the two channels. For example, if a microphone is connected to the right input channel and
 an instrument is connected to the left input channel which is much louder than the microphone, move the audio fader to increase the relative volume of the mic.
 
-# Menu commands
-
-### File > Load/Save Mixer Channels Setup
- 
-You can save and restore the mix you have for your band rehearsals (fader, mute, pan, solo etc.) and load these any time (even while you are playing). Loading can also be done by drag/drop to the mixer window.
-
-### Edit > Auto-Adjust All Faders 
-
-Applies a one-off fader setting to each channel depending on its volume. Useful for large ensembles to get a reasonable overall mix, although individual adjustments might still be necessary. Best applied during a warm-up or a uniform part of the music piece.
-
 # Backing up Jamulus
 
 {% include_relative Include-Backing-Up.md %}
 * You can save and load different mixer settings using [Load/Save Mixer Channels Setup](Software-Manual#file--loadsave-mixer-channels-setup) and store those files wherever you want.
 
-**Note for macOS users:** As of Jamulus 3.8.1, we have a signed installer. This will store the settings in 
+**Note for macOS users:** As of Jamulus 3.8.1, we have a signed installer. This will store the settings in
```

Comment:
Probably we should get rid of the as of ... part.

Reply:
Yep, not here though.  Just cleaning up trailing spaces.

Reply:
Raised https://github.com/jamulussoftware/jamuluswebsite/issues/1062 for this.